EIPS/eip-7775.md

```diff
@@ -0,0 +1,100 @@
+---
+eip: 7775
+title: New opcode BURN
+description: An opcode to burn native token at the given address
+author: Dev Bear (@itsdevbear)
+discussions-to: <URL>
```

Comment:
Please create a topic on Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5

Reply:
@abcoathup when I try to post:

![image](https://github.com/user-attachments/assets/fc3faf06-40f1-48e9-8bef-3627dccc491e)


any suggestions?

Reply:
Not sure why you can't access Eth Magicians: https://ethereum-magicians.org

Can you try again. If you haven't already created an account you will need to read for a little while before you can create a new topic.